test/test_consumer_integration.py

```diff
@@ -88,6 +94,12 @@ def kafka_consumer(self, **configs):
                                  **configs)
         return consumer
 
+    def kafka_producer(self, **configs):
+        brokers = '%s:%d' % (self.server.host, self.server.port)
+        producer = KafkaProducer(
+            bootstrap_servers=brokers, **configs)
+        return producer
```

Comment:
I wonder if this producer should just get moved to a `pytest` fixture. Not something that needs to be handled in this PR, but should probably create an issue saying we should clean it up.